### DISCUSSION TITLE
Compatibility between Theia frontend and backend between different versions

### DISCUSSION THREAD

**Discussion by viralmodi** [eclipse-theia/theia#8934] [General] [answered]
Does Theia make any guarantees of compatiblity between Theia frontend and backend between different versions of Theia?

**Comment by vince-fugnitto** [comment, marked as answer]
@viralmodi in order to successfully build and package the application, all `@theia` dependencies must be the exact same version. It is currently not possible to mix-and-match individual extension versions of the framework.
